scripts/bin/common

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
We don't need a shebang (or `+x`) if it's just being sourced.

Reply:
I used a shebang rather than have vim magic at the top to set the filetype, since it's more universal.

Reply:
What do I do when a person exhibits undefined behavior